packages/plugin-credentials/src/credentials/browser.js

```diff
@@ -147,6 +152,8 @@ const Credentials = SparkPlugin.extend(Object.assign({}, common, {
         this._updateLocation(location);
       }
 
+      // this should be set as false since we are no more validating the user for credentials. That means if the user already has credentials in his localStorage then he should be able to login without being presented the login screen.
+      this.isValidatingUser = false;
```

Comment:
Does this also need to happen in a `.then` block attached to `requestAuthorizationCodeGrant()` up above?